src/pages/components/text.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { NamespacesConsumer } from 'react-i18next';
+import { withI18next } from '@wapps/gatsby-plugin-i18next';
+
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import ComponentDescription from 'components/ComponentDescription';
+import sideNavData from 'config/sidenav/components';
+import NotificationBox from 'components/NotificationBox';
+import Section from 'components/Section';
+import ComponentExample from 'components/ComponentExample';
+import { Heading, Paragraph } from 'components/ResponsiveComponents';
+import { Text } from 'components/ExampleComponents';
+
+const Page: React.FC = (): React.ReactElement => {
+  return (
+    <NamespacesConsumer ns={['text']}>
+      {t => (
+        <Layout sideNavData={sideNavData(t)}>
+          <SEO title={t('title')} />
+          <Heading.h1>{t('title')}</Heading.h1>
+
+          <Paragraph.lead>
+            <Text.lead>{t('intro')}</Text.lead>
+          </Paragraph.lead>
+
+          <NotificationBox />
+
+          {t('sections').map((section, index) => (
+            <Section
+              key={index}
+              mainTitle={section.title}
+              paragraphs={section.paragraphs}
+              links={section.links}
+            />
+          ))}
+
+          <ComponentDescription
+            mainTitle={t('body.title')}
+            description={t('body.description')}
+            exampleFirst={false}
+            filterProps={[]}
+          >
+            <ComponentExample>
+              <div
+                style={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  flex: 1,
+                }}
+              >
+                <Text>Body text</Text>
+                <Text.bold>Body Text Bold</Text.bold>
+                <Text smallScreen>Body Text Small</Text>
+                <Text.bold smallScreen>Body Text Small Bold</Text.bold>
+              </div>
+            </ComponentExample>
+          </ComponentDescription>
+
+          <ComponentDescription
+            mainTitle={t('lead.title')}
+            description={t('lead.description')}
+            exampleFirst={false}
+            filterProps={[]}
+          >
+            <ComponentExample>
+              <div
+                style={{
```

Comment:
Is this something that repeats for every component? While not wrong in any way, I'd like to watch out for things we end up duplicating in our code, even if the duplication happens in very small scale for the time being.

I think we might be gaining technical debt in this repo and should start planning improvements and refactoring.

Reply:
What exactly are you referring to?

Reply:
I now committed change where the is new small helper function (_ExampleBlock_) to prevent duplication.

Reply:
@LJKaski I was referring to small things here and there. Custom components maintained over the ones we have available in ui-components, extensive use of inline styles in components (why not use styled components?), inconsistent and mixed use of design tokens and fixed css styles and so on. I guess it's more of a gut feeling for not having a clear plan for improvements and having quite a few things not exactly "finished" or done quick and dirty => maintenance will become cumbersome at some point.